frontend/frontend.go

```diff
@@ -161,8 +173,20 @@ func (f *Frontend) ArmResourceListBySubscription(writer http.ResponseWriter, req
 
 func (f *Frontend) ArmResourceListByLocation(writer http.ResponseWriter, request *http.Request) {
 	ctx := request.Context()
-	logger := ctx.Value(ContextKeyLogger).(*slog.Logger)
-	versionedInterface := ctx.Value(ContextKeyVersion).(api.Version)
+	logger, ok := LoggerFromContext(ctx)
+	if !ok {
+		logger = DefaultLogger()
```

Comment:
I believe this is the same example. If there is no proper context, error and abort.

Reply:
Yes, done. Thanks!